apps/test/util/runTests.js

```diff
@@ -77,7 +77,8 @@ exec(command, function (err, stdout, stderr) {
     transform: 'ejsify',
     'web-security': false
   })
- .on('error', function () {
+ .on('error', function (e) {
+   console.log(e.stack);
```

Comment:
It does, with slightly more verbose output.  Is that preferred?  Neither seems to be standard.

Here's `console.trace()` output:
```bash
╰○ grunt mochaTest
Running "exec:mochaTest" (exec) task
>> Error: Exit 127
>> Trace
>>     at Browserify.<anonymous> (/home/brad/Projects/code-dot-org/apps/test/util/runTests.js:82:12)
>>     at Browserify.emit (events.js:129:20)
>>     at /home/brad/Projects/code-dot-org/apps/node_modules/mochify/lib/phantom.js:29:11
>>     at /home/brad/Projects/code-dot-org/apps/node_modules/mochify/node_modules/phantomic/lib/phantomic.js:141:9
>>     at null._onTimeout (/home/brad/Projects/code-dot-org/apps/node_modules/mochify/node_modules/phantomic/lib/phantomic.js:59:11)
>>     at Timer.listOnTimeout (timers.js:119:15)
>> Exited with code: 1.
```

And here's `console.log(error.stack)`:
```bash
╰○ grunt mochaTest
Running "exec:mochaTest" (exec) task
>> Error: Exit 127
Error: Exit 127
    at /home/brad/Projects/code-dot-org/apps/node_modules/mochify/lib/phantom.js:29:25
    at /home/brad/Projects/code-dot-org/apps/node_modules/mochify/node_modules/phantomic/lib/phantomic.js:141:9
    at null._onTimeout (/home/brad/Projects/code-dot-org/apps/node_modules/mochify/node_modules/phantomic/lib/phantomic.js:59:11)
    at Timer.listOnTimeout (timers.js:119:15)
>> Exited with code: 1.
```